IanGRap/Squish
# Cannon block model

Contributors: Ian

Goal: Add the legitimate model to the cannon block

Important Files:
- CannonPusher.cs: Removed all code that would change the UI that showed the number of uses remaining for the cannon block. Instead, the color of the crystal changes to represent usage.

How to test:
- Play the match scene a few times. Eventually you should see the cannon block appear as a hazard with its model. The color of the crystal should change as you push it. Should the colors be changed? Should the text that used to show how many pushes were remaining be put back in?
